packages/api/src/services/jobPosts.service.ts

```diff
@@ -6,18 +6,16 @@ import {
   type JobPostsSchemaInsert,
 } from "../models/jobs.model";
 import { and, eq, isNull } from "drizzle-orm";
-import { AppError } from "../utils/ExpressError";
+import { AppError } from "../utils/express.error";
+import { jobPostsSchema } from "../rules/jobPost.rule";
+import z from "zod";
 
-export function normalizeJobPost(
-  job: JobPostsSchema,
-  companyId?: number
-): JobPostsSchema {
+export type job = Required<z.infer<typeof jobPostsSchema>>;
```

Comment:
Convention is to use capital letters for the first char in a types name:

```suggestion
export type Job = Required<z.infer<typeof jobPostsSchema>>;
```